nearlib/account.js

```diff
@@ -1,6 +1,6 @@
 const bs58 = require('bs58');
 
-const { CreateAccountTransaction, SignedTransaction } = require('./protos');
+const { google, AccessKey, AddKeyTransaction, CreateAccountTransaction, SignedTransaction } = require('./protos');
```

Comment:
do we need to add the new classes to index.js and browser.js?

Reply:
No, since we don't return them to the browser or index